.github/actions/sphinx/deploy/action.yml

```diff
@@ -12,7 +12,8 @@ runs:
       run: |
         git config user.name 'github-actions[bot]'
         git config user.email 'github-actions[bot]@users.noreply.github.com'
+        git config commit.cleanup 'verbatim'
 
         git add docs/latest
-        git commit -m 'docs: Build documentation triggered by ${{ github.sha }}'
+        git commit -m $'docs: Build documentation triggered by ${{ github.sha }}\n\n\nskip-checks:true'
```

Comment:
What is the `$` for?

Reply:
It's a [bash feature](https://www.gnu.org/software/bash/manual/html_node/ANSI_002dC-Quoting.html) used to add newline characters (and more).

Reply:
TIL, thanks!